prebid-mobile/prebid-mobile-rendering.md

```diff
@@ -0,0 +1,110 @@
+---
+
+layout: page_v2
+title: Rendering Module
+description: Description of how Prebid SDK supports rendering
+sidebarType: 2
+
+---
+
+# Rendering Module Overview
+
+> ⚠️ NOTE: Rendering is an Open Beta Feature
+
+Prebid SDK introduces the rendering facilities as a standalone module. Adding this module won't change anything in your current integration. 
+
+![How Prebid Mobile Works]({{site.baseurl}}/assets/images/prebid-mobile/rendering/modularization-roadmap-open-beta.png){: .pb-lg-img :}
+
+In the scope of the Open Beta release, you can use the Core and Rendering modules alongside each other. The current integration will work as is without changes but if you want to add or switch to the rendering integration you will have to make changes in the codebase.  
+
+The rendering module consist of three libraries:
+
+- **PrebidSDKRendering**, which introduces the classes for integration into UI and rendering engine
+- [OPTIONAL] **PrebidSDKGAMEventHandlers**, which wraps GAM classes in order to support GAM integration. 
+- [OPTIONAL] **PrebidSDKMoPubAdapters**, which wraps MoPub Adapters for Prebid Rendering module
+
+Using rendering module you can integrate next ad formats:
+
+- **Dispaly** - Banner/Interstitial
+- **Outsrteam Video** - Banner/Interstitial/Rewarded
+- **Native** - Disaply / Video
+
+Intefration steps are convinient and described in the respective docs for [iOS](TBD) and [Android](TBD). 
+
+## Motivation and Benefits 
+
+Rendering serves the general Prebid's aim - to be a trusted, transparent, and configurable monetization solution and gives a lot of benefits to both sides publishers and demand partners. 
+
+In comparising with no-rendering approach the new module gives next benefits:
+
+- Monitizaion without Ad Server. Publishers can monitize the apps through pure In-App bidding without additional ad requests.
+- Faster ad delivery. As soon as bid is recieved it can be displayed. It is applayed to both kind of integration with ad server  or without it.
```

Comment:
- Reduced ad delivery latency: With the Prebid rendering module, Prebid SDK can render ads immediately when demand is returned from Prebid Server or when receiving the render signal from an ad server. The full render process should vastly reduce ad delivery speeds.